CHANGES.rst

```diff
@@ -2289,6 +2289,10 @@ tweakreg
 
 - Updated step arguments in the documentation. [#4723]
 
+- Make ``fit_quality_is_good()`` member private and rename it to
```

Comment:
The location of this entry in the change log puts it way down into the section for a very old release. It should be moved to the top section under "unreleased".

Reply:
My bad

Reply:
Fixed changelog